src/components/ActionSheet/index.stories.tsx

```diff
@@ -0,0 +1,250 @@
+      value: 1,
+      callback: (item: ActionItem) => {
+        Toast.info({ message: item.name });
+        setVisibleTitle(false);
+      }
+    }
+  ];
+  const actionsCancel: ActionItem[] = [
+    {
+      name: 'Option',
+      value: 1,
+      callback: (item: ActionItem) => {
+        Toast.info({ message: item.name });
+        setVisibleTitle(false);
+      }
+    }
+  ];
+  const actionsCancelIcon: ActionItem[] = [
+    {
+      name: 'Option',
+      value: 1,
+      callback: (item: ActionItem) => {
+        Toast.info({ message: item.name });
+        setVisibleTitle(false);
+      }
+    }
+  ];
+  const actionTitle: ActionItem[] = [
+    {
+      name: 'Option',
+      value: 1,
+      callback: (item: ActionItem) => {
+        Toast.info({ message: item.name });
+        setVisibleTitle(false);
+      }
+    }
+  ];
+  const actionsDescription: ActionItem[] = [
+    {
+      name: 'Option',
+      value: 1,
+      callback: (item: ActionItem) => {
+        Toast.info({ message: item.name });
+        setVisibleTitle(false);
+      }
+    }
+  ];
+  return (
+    <div className='storybook__container column grey'>
+      <Cell
+        {...{
+          title: {
+            text: 'Title',
+            fontSize: '12px'
+          },
+          onClick: () => {
+            setVisibleTitle(true);
+          }
+        }}
+      />
+      <Cell
+        {...{
+          title: {
+            text: 'Description',
+            fontSize: '12px'
+          },
+          onClick: () => {
+            setVisibleDescription(true);
+          }
+        }}
+      />
+      <Cell
+        {...{
+          title: {
+            text: 'CancelText',
+            fontSize: '12px'
+          },
+          onClick: () => {
+            setVisibleCancel(true);
+          }
+        }}
+      />
+      <Cell
+        {...{
+          title: {
+            text: 'CancelIcon',
+            fontSize: '12px'
+          },
+          onClick: () => {
+            setVisibleCancelIcon(true);
+          }
+        }}
+      />
+      <ActionSheet
+        visible={visibleTitle}
+        maskClick={() => setVisibleTitle(false)}
+        actions={actionTitle}
+        title='This is title'
+      />
+      <ActionSheet
+        visible={visibleDescription}
+        maskClick={() => setVisibleDescription(false)}
+        actions={actionsDescription}
+        description='This is description'
+      />
+      <ActionSheet
+        visible={visibleCancel}
+        cancelText='CancelButton'
+        cancelClick={() => setVisibleCancel(false)}
+        actions={actionsCancel}
+      />
+      <ActionSheet
+        visible={visibleCancelIcon}
+        cancelText='CancelButton'
+        cancelClick={() => setVisibleCancelIcon(false)}
+        closeable
+        actions={actionsCancelIcon}
+      />
+      <ActionSheet
+        visible={visibleCancelIcon}
+        cancelText='CancelButton'
+        cancelClick={() => setVisibleCancelIcon(false)}
```

Comment:
could you rename this to `onCancel`? to stay consistent with the rest of the API